lib/travis/build/script/dart.rb

```diff
@@ -43,24 +83,46 @@ def setup
             ansi: :green
           sh.echo '  https://github.com/travis-ci/travis-ci/issues' \
             '/new?labels=community:dart', ansi: :green
-          sh.echo 'and mention \`@a14n\`, \`@devoncarew\` and \`@sethladd\`' \
+          sh.echo 'and mention `@nex3` and \`@a14n\`' \
```

Comment:
`\` are missing and leads to [the following error in the job](https://staging.travis-ci.org/BanzaiMan/travis_staging_test/jobs/545217#L119).

>/home/travis/build.sh: line 440: @nex3: command not found